docs/specifications/url.rst

```diff
@@ -40,7 +40,7 @@ NURLs
 
 The authentication and authorization properties of fURLs are a good fit for Tahoe-LAFS' requirements.
 These are not inherently tied to the Foolscap protocol itself.
-In particular they are beneficial to :doc:`../proposed/http-storage-node-protocol` which uses HTTP instead of Foolscap.
+In particular they are beneficial to :doc:`../specifications/http-storage-node-protocol` which uses HTTP instead of Foolscap.
```

Comment:
hmm, might be simpler if this is just :doc:`http-storage-node-protocol` since it's in the right directory already?